Resources/Prototypes/Nyanotrasen/Entities/Mobs/Player/robots.yml

```diff
@@ -293,10 +293,10 @@
   - type: Laws
```

Comment:
the language is directly adapted from various hippocratic oaths, I don't really think any language changes are needed here